Update give Cloud access to a GCP user (#1087)
RedisLabs/redislabs-docs: content/rc/administration/account-team-settings.md

### Team management for GCP Marketplace customers

If you subscribed to Redis Enterprise Cloud using GCP Marketplace, you can manage your team from the **IAM section** of the GCP console.

> To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and assign one of these roles to the user:

Reviewer: ```suggestion
To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and then assign one of the following roles to the user:
```

 serviceusage.serviceUsageViewer & redisenterprisecloud.viewer - these roles should be assigned to member designated as viewer
 serviceusage.serviceUsageViewer & redisenterprisecloud.admin -  these roles should be assigned to member designated as owner
 
In case these roles do not appear, add a role to your project by doing the following:
1. Select **Manage Roles** 
1. Search the role on the **filter table** field ( for example: "service usage viewer" or "redis enterprise cloud admin")

> 1. Mark the checkbox

Reviewer: ```suggestion
1. Place a check in the checkbox
```

1. Select **Create role from selection** and select **Create**
1. Go back to **IAM** to add a member and assign the desried roles.

> Users must log in using SSO to Redis Cloud at least once for them to be added to the team.

Reviewer: ```suggestion
Users are not added to the team until they sign in to Redis Cloud using their single-sign on (SSO) credentials.
```

> This cam be done by clicking on the button **manage on provider** 

Reviewer: ```suggestion
Use the **manage on provider** button to do so. 
```
